tests/KotlinJava17Test.php

```diff
@@ -17,7 +17,7 @@ class KotlinJava17Test extends Base
         'chmod +x tests/sdks/kotlin/gradlew',
     ];
     protected string $command =
-        'docker run --network="mockapi" -v $(pwd):/app -w /app/tests/sdks/kotlin openjdk:17-jdk-slim sh -c "./gradlew test -q && cat result.txt"';
+        'docker run --network="mockapi" --rm -v $(pwd):/app -w /app/tests/sdks/kotlin openjdk:17-jdk-slim sh -c "./gradlew test -q && cat result.txt"';
```

Comment:
Can you explain why was this change required?

Reply:
It's just cleanup, so I can safely re-run tests without having deleting container manually. I see the same in other tests too.